test/uk/gov/hmrc/trustsstore/controllers/MaintainTaskListControllerSpec.scala

```diff
@@ -70,7 +70,7 @@ class MaintainTaskListControllerSpec extends BaseSpec {
   "invoking PUT /maintain/task/trustees" - {
```

Comment:
@adamconder @JoshuaH01 test heading has PUT in it,  should be POST now?

Reply:
Well spotted I'll get this fixed 👍

Reply:
Raised a pull request to resolve it https://github.com/hmrc/trusts-store/pull/16 😎